### ISSUE TITLE
NCMC bug

### ISSUE THREAD

**Issue by jchodera** [choderalab/perses#16]
This line:
https://github.com/pgrinaway/perses/blob/run_example/perses/annihilation/ncmc_switching.py#L279
```python
        self.addComputeGlobal('initial_total_energy', 'ke + potential')
```
should be
```python
        self.addComputeGlobal('final_total_energy', 'ke + potential')
```

**Comment by pgrinaway** [comment]
Ok, fixed by pushing to #14 , but it looks like `ncmc_elimination_logp` and `ncmc_insertion_logp` are both 0.0... investigating further...

**Comment by jchodera** [comment]
Thanks!
